orderbook/openapi.yml

```diff
@@ -155,6 +155,21 @@ paths:
                 $ref: "#/components/schemas/FeeInformation"
         400:
           description: sellToken not existent
+  /api/v1/solvable_orders:
```

Comment:
Should we somehow separate the core and extended api in the path? Maybe by `/core/v1/solvable_orders` or `api/v1/core/solvable_orders`?

Reply:
Not sure. If we keep everything in the same api docs then also feels like it doesn't need separate paths and these categories like "core" feel like something we do internally but doesn't have to show on the api.